From 7d5cfec8b2040a6f1bdfacf190e06578e379df79 Mon Sep 17 00:00:00 2001
From: Octavian Purdila <octavian.purdila@nxp.com>
Date: Mon, 13 Feb 2017 15:17:26 +0200
Subject: [PATCH 1400/5242] MLK-13922: ARM: dts: imx6qp-sabreauto: add
 lvd-channel info to ldb

commit  134de616f2f706a09cf3754d4aaa3a181d057dac from
https://source.codeaurora.org/external/imx/linux-imx.git

Also, enable the mxcfb* entries.

Signed-off-by: Octavian Purdila <octavian.purdila@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx6qp-sabreauto.dts |   14 ++++++++++++++
 1 file changed, 14 insertions(+)

diff --git a/arch/arm/boot/dts/imx6qp-sabreauto.dts b/arch/arm/boot/dts/imx6qp-sabreauto.dts
index 4bee45f..9137fde 100644
--- a/arch/arm/boot/dts/imx6qp-sabreauto.dts
+++ b/arch/arm/boot/dts/imx6qp-sabreauto.dts
@@ -12,20 +12,34 @@
 	compatible = "fsl,imx6qp-sabreauto", "fsl,imx6qp";
 };
 
+&ldb {
+	lvds-channel@0 {
+		crtc = "ipu2-di0";
+	};
+
+	lvds-channel@1 {
+		crtc = "ipu2-di1";
+	};
+};
+
 &mxcfb1 {
 	prefetch;
+	status = "okay";
 };
 
 &mxcfb2 {
 	prefetch;
+	status = "okay";
 };
 
 &mxcfb3 {
 	prefetch;
+	status = "okay";
 };
 
 &mxcfb4 {
 	prefetch;
+	status = "okay";
 };
 
 &fec {
-- 
1.7.9.5

